generator-test/src/test/pegasus/com/linkedin/pegasus/generator/test/idl/escaping/PropertyKeyEscaping.pdl

```diff
@@ -2,6 +2,7 @@ namespace com.linkedin.pegasus.generator.test.idl.escaping
 
 record PropertyKeyEscaping {
   @`namespace` = "foo.bar"
+  @path.`/*.*/`
```

Comment:
I'd suggest buffing this test case up a little bit to test the following things:
1. All types of comments (e.g. single-line, javadoc):
```
@`//`.`/**`.`*/`.`/*.*/`
```
2. Test the encoder pattern you just added (`Pattern.compile(".*[^0-9a-zA-Z_-].*")`):
```
@shouldNotBeEscaped.abc_XYZ-123
```

Reply:
Done, improved the grammar to handle non-identifier but should not be escaped property segments